sessiontxn/failpoint.go

```diff
@@ -94,6 +98,20 @@ func AssertTxnManagerReadTS(sctx sessionctx.Context, expected uint64) {
 	}
 }
 
+// AddEntranceForLockError is used only for test
+func AddEntranceForLockError(sctx sessionctx.Context, name string) {
```

Comment:
AddEntranceForLockError => AddAssertEntranceForLockError may be better

Reply:
Done